### PR TITLE
Feature/multiple view

### PR DESCRIPTION
**Pull request by grzanka**

### CODE REVIEW

**Comment by grzanka** [comment]
@ostatni5 can you rebase this one on latest master ?

File: src/ThreeEditor/util/CSGManager.ts [hunk 1]
```diff
@@ -162,7 +162,7 @@ class CSGManager {
         this.editor.sceneHelpers.remove(zone.resultMesh);
         this.zones.delete(zone.uuid);
     }
-
+
```
Comment (Derstilon) [inline src/ThreeEditor/util/CSGManager.ts:165]:
```suggestion
```

File: src/ThreeEditor/js/Menubar.Layout.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	option.setClass( 'option' );
+	option.setTextContent( 'Four Views' );
+	option.onClick( function () {
+
+		editor.signals.layoutChanged.dispatch('fourViews');
+
+	} );
+	options.add( option );
+
+
```
Comment (Derstilon) [inline src/ThreeEditor/js/Menubar.Layout.js:43]:
```suggestion
	function createOption(optionClass, optionText, optionClick){
		let option = new UIRow();
		option.setClass(optionClass);
		option.setTextContent(optionText);
		option.onClick(optionClick);
		return option;
	}

	// Single view

	options.add( 
		createOption(
			'option', 
			'Single View',
			()=>{
				editor.signals.layoutChanged.dispatch('singleView');
			}, 
		)
	);

	// Four view
	
	options.add( 
		createOption(
			'option', 
			'Four Views',
			()=>{
				editor.signals.layoutChanged.dispatch('fourViews');
			}, 
		)
	);
```

**Review by Derstilon — APPROVED** [review]
I've added few improvements and cleaned up the code. Apart from that everything you made works just fine

**Pull request merged**